GrandCentralBoard/App/ViewControllers/LoadingViewController.swift

```diff
@@ -18,7 +18,8 @@ class LoadingViewController: UIViewController {
         BonusWidgetBuilder(dataDownloader: self.dataDownloader),
         GoogleCalendarWatchWidgetBuilder(),
         HarvestWidgetBuilder(),
-        ImageWidgetBuilder(dataDownloader: self.dataDownloader)
+        ImageWidgetBuilder(dataDownloader: self.dataDownloader),
+        WebsiteAnalyticsWidgetBuilder()
```

Comment:
I am not sure, we wanted more generic name, because this widget can show any number of values for given pages. So we decided for `WebsiteAnalyticsWidget` - we show blog posts, but this is just our use case. With different data source it could be sth else.